src/state/controls/Control.spec.ts

```diff
@@ -58,4 +59,26 @@ describe('control', () => {
         });
         stub.restore();
     });
+
+    it('allows batch updates', () => {
+        const buttonDiff: IButtonUpdate = {
+            cost: 200,
+            text: 'foobar',
+        };
+        const updatedButton = {
+            ...{
```

Comment:
you don't need to spread an object on here, you can include the properties directly